auth/database/historicModels.py

```diff
@@ -18,10 +19,10 @@ class UserInactive(db.Model):
     __tablename__ = 'user_inactive'
 
     id = Column(Integer, primary_key=True, autoincrement=False)
-    name = Column(String, nullable=False)
-    username = Column(String(50), nullable=False)
-    service = Column(String, nullable=False)
-    email = Column(String, nullable=False, unique=True)
+    name = Column(String(UserLimits.name), nullable=False)
+    username = Column(String(UserLimits.username), nullable=False)
+    service = Column(String(UserLimits.service), nullable=False)
+    email = Column(String(UserLimits.email), nullable=False, unique=False)
```

Comment:
why drop the constraint?

Reply:
If the automated test script is ran twice, some users are created and deleted twice.
The second user can be created, but the unique constraint on the e-mail field on the inactive table will deny the deletion of the user.
 
if we want unique e-mail and username across active and inactive users, uniqueness checks 
must be done on both tables on user creation. Also, a 'restore inactive user' feature is necessary to reuse a user e-mail.

Anyway, I think those uniqueness checks should be added in another PR.

Reply:
mmm :thinking:  

I have a feeling we should discuss the particulars of why we're keeping this users history.
If that is just for logging purposes, perhaps keeping that information on a relational database seems like throwing too much resources at a lesser problem. If not, then those checks definitely need to be implemented. 

That said, I agree with you, we can leave that for later. I've created a new issue (#17) to track that up.